### ISSUE TITLE
Multiple Delete in data option

### ISSUE THREAD

**Issue by kellyho67** [rlightner/public-BrewStatus#50]
For some reason my tilt was logging data to my new associated beer as well as the previous one.. I had to delete about 10 pages of data in the old beer.. would be cool sometime if you could add a multi select delete option.. I know this is not anything urgent just an idea..

**Comment by rlightner** [comment]
But in the settings, it only showed it being associated with one Brew?

**Comment by kellyho67** [comment]
That is correct, I was only able to validate with the same time stamps.. I do not see it doing the same thing now nor do I see any updates from the time I deleted it vs re-associated it.

Thanks
Kelly
